### PR TITLE
[MXNET-696] Fix undefined variable errors

### PR DESCRIPTION
**Pull request by vandanavk**
## Description ##
Pylint throws an error for variables that are used but not defined previously. Since these variables are found in paths that are not executed, they don't show up as errors at runtime. But if executed, they could throw a Python NameError.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- image_segmentation.py - Defining ctx to be cpu by default.
- coco.py - maskUtils is disabled as of now. Commenting out the code that references this until it is implemented.
- common.py - start_offset was uninitialized. Setting this variable to a default of 0.1 based on information from the author of the example and the reference paper.

## Comments ##
- Related issues https://github.com/apache/incubator-mxnet/issues/8270, https://github.com/apache/incubator-mxnet/issues/11904

@cclauss @zhreshold

### CODE REVIEW

**Comment by vandanavk** [comment]
@marcoabreu Yes, thats the plan. But it will take some effort - the fixes in these commits are only for one category of Pylint errors (undefined-variable). Some of the examples have not had Pylint executed on them - so they have errors such as line-too-long etc, which are normally found for python/mxnet/ by the CI build.

**Comment by vandanavk** [comment]
@apeforest @sandeep-krishnamurthy

File: example/ssd/dataset/pycocotools/coco.py [hunk 1]
```diff
@@ -255,22 +255,23 @@ def showAnns(self, anns):
...
-                        t = self.imgs[ann['image_id']]
-                        if type(ann['segmentation']['counts']) == list:
+                        raise NotImplementedError("maskUtils disabled!")
+                        # t = self.imgs[ann['image_id']]
+                        # if type(ann['segmentation']['counts']) == list:
                             # rle = maskUtils.frPyObjects([ann['segmentation']], t['height'], t['width'])
-                            raise NotImplementedError("maskUtils disabled!")
-                        else:
-                            rle = [ann['segmentation']]
+                        #    raise NotImplementedError("maskUtils disabled!")
```
Comment (apeforest) [inline example/ssd/dataset/pycocotools/coco.py]: If these are not needed, I would prefer just removing them instead of commenting out.

**Comment by marcoabreu** [comment]
Definitely! Thanks a lot for your efforts!

>

**Comment by marcoabreu** [comment]
Sorry, I'm currently swamped with high priority tasks and I don't have time to review your task or assist with inqueries.

**Pull request merged**